full/src/main/java/apoc/uuid/UUIDHandlerNewProcedures.java

```diff
@@ -0,0 +1,118 @@
+package apoc.uuid;
+
+import apoc.ApocConfig;
+import apoc.SystemLabels;
+import apoc.SystemPropertyKeys;
+import apoc.util.Util;
+import org.neo4j.graphdb.Label;
+import org.neo4j.graphdb.Node;
+import org.neo4j.graphdb.ResourceIterator;
+import org.neo4j.graphdb.Transaction;
+import org.neo4j.graphdb.schema.ConstraintDefinition;
+import org.neo4j.graphdb.schema.Schema;
+import org.neo4j.internal.helpers.collection.Pair;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Stream;
+import java.util.stream.StreamSupport;
+
+import static apoc.ApocConfig.*;
+import static apoc.SystemPropertyKeys.*;
+import static apoc.SystemLabels.*;
+import static apoc.util.SystemDbUtil.getSystemNodes;
+import static apoc.util.SystemDbUtil.setLastUpdate;
+import static apoc.util.SystemDbUtil.withSystemDb;
+import static apoc.uuid.UuidHandler.NOT_ENABLED_ERROR;
+
+public class UUIDHandlerNewProcedures {
+    public static boolean isEnabled(String databaseName) {
+        String apocUUIDEnabledDb = String.format(ApocConfig.APOC_UUID_ENABLED_DB, databaseName);
+        return apocConfig().getConfig().getBoolean(apocUUIDEnabledDb, apocConfig().getBoolean(APOC_UUID_ENABLED));
```

Comment:
what happens if apoc.uuid.enabled is not set, will it fail to start the db? like we had recently?